pom.xml

```diff
@@ -66,6 +66,12 @@
             <artifactId>duckdb_jdbc</artifactId>
             <version>0.7.1</version>
         </dependency>
+        <dependency>
```

Comment:
Can you move this dependency (as well as duckdb) to different profiles? That way we can enabling packaging the necessary drivers for a certain setup with -P.